app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -87,6 +106,22 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
    **/
   private var alreadyFocusedOnEmptyTextField: Boolean = false
 
+  private val viewRenderer = PatientEntryViewRenderer(this)
+
+  private val events = PublishSubject.create<PatientEntryEvent>().hide().share()
```

Comment:
We don't need to `share()` this stream since it has only one consumer. Did you mean to also replace the event stream going to the controller with this shared stream?

Reply:
Discussed offline: This will break the feature because the stream is not shared between the old controller and the new mobius loop.

Reply:
Fixed this.